packages/trace-viewer/src/ui/watchMode.css

```diff
@@ -68,20 +62,19 @@
 }
 
 .status-line {
-  flex: none;
+  flex: auto;
```

Comment:
New status line is on top.
<img width="247" alt="Screenshot 2023-03-18 at 8 22 07 PM" src="https://user-images.githubusercontent.com/883973/226151839-b4422ad9-e426-484b-b8da-7d69d2584275.png">